src/python_extension/cps_example.py

```diff
@@ -21,7 +21,7 @@
 if __name__ == '__main__':
     # load the mapping file
     cps.init(
-        '/localdisk/cwichmann/cps-api/workspace/debian/jessie/x86_64/sysroot/opt/dell/os10/lib/',
+        '/localdisk/cwichmann/cps-api/workspace/debian/jessie/x86_64/sysroot/usr/lib/',
```

Comment:
Instead of a hard-coded path, perhaps we should change this example to take a command line argument for the directory containing libcpsclass shared objects.